cinema_service/settings.py

```diff
@@ -137,3 +137,9 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+REST_FRAMEWORK = {
+    "DEFAULT_PERMISSION_CLASSES": [
+        "rest_framework.permissions.AllowAny",
+    ]
+}
```

Comment:
The `DEFAULT_PERMISSION_CLASSES` is set to `AllowAny`, which means all API endpoints are accessible without authentication. Ensure this is intentional and aligns with your security requirements, especially in production environments.